CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.1.0]
+### Changed
+- Update `eth-block-tracker` from v5 to v6 ([#188](https://github.com/MetaMask/eth-json-rpc-middleware/pull/188))
+ - Breaking changes do not affect this package.
```

Comment:
Seems like this might be missing an extra space (this isn't indented currently).